README.md

```diff
@@ -2,19 +2,22 @@
 
 Solarkraft is a runtime monitoring tool for [Soroban][], powered by [TLA+][] and [Apalache][].
 
-The project is in the boostrapping phase now. Preparing for Warp 5.
-
-We are grateful to the [Stellar Community Fund][] for supporting our project via
-the Activation Award. Check our [3-minute pitch][], and the [project details](#project-details) below.
-
-![activation award](./assets/solarkraft-stellar-activation.png)
+We have finished the activation phase and developed an MVP. See a
+[demo](https://youtu.be/E99TNzTHLXI) by Jure Kukovec.
```

Comment:
Wouldn't it be better to link to the blogpost with both videos embedded?

Reply:
Good point, just added it.